package.json

```diff
@@ -25,6 +25,11 @@
         "require": "./dist/index.js",
         "import": "./wrapper.mjs"
       },
+      "browser": {
+        "import": "./dist/esm-browser/index.js",
+        "require": "./dist/index.js",
+        "default": "./dist/esm-browser/index.js"
```

Comment:
(Why) is this default condition needed?

Reply:
Just to have a fallback if neither of `require` or `import` is provided. Reading https://nodejs.org/api/packages.html#nested-conditions that's not needed tho